label_studio_ml/examples/gliner/README.md

```diff
@@ -0,0 +1,60 @@
+The GLiNER model is a BERT family model for generalist NER. We download the model from HuggingFace, but the original
+model is
+available on [GitHub](https://github.com/urchade/GLiNER).
+
+
+## Running with Docker (Recommended)
+
+1. Start Machine Learning backend on `http://localhost:9090` with prebuilt image:
+
+```bash
+docker-compose up
+```
+
+2. Validate that backend is running
+
+```bash
+$ curl http://localhost:9090/
+{"status":"UP"}
+```
+
+3. Connect to the backend from Label Studio running on the same host: go to your project `Settings -> Machine Learning -> Add Model` and specify `http://localhost:9090` as a URL.
+
+
+## Building from source (Advanced)
+
+To build the ML backend from source, you have to clone the repository and build the Docker image:
+
+```bash
+docker-compose build
+```
+
+## Running without Docker (Advanced)
+
+To run the ML backend without Docker, you have to clone the repository and install all dependencies using pip:
+
+```bash
+python -m venv ml-backend
+source ml-backend/bin/activate
+pip install -r requirements.txt
+```
+
+Then you can start the ML backend:
+
+```bash
+label-studio-ml start ./dir_with_your_model
+```
+
+# Configuration
+Parameters can be set in `docker-compose.yml` before running the container.
+
+
+The following common parameters are available:
+- `BASIC_AUTH_USER` - specify the basic auth user for the model server
+- `BASIC_AUTH_PASS` - specify the basic auth password for the model server
+- `LOG_LEVEL` - set the log level for the model server
+- `WORKERS` - specify the number of workers for the model server
+- `THREADS` - specify the number of threads for the model server
+- `LABEL_STUDIO_URL` - specify the URL of your label studio instance. Note that this might need to be `http://host.docker.internal:8080` if running Label Studio on another Docker container 
+- `LABEL_STUDIO_API_KEY`- specify the API key for authenticating your label studio instance 
+- `LABELS` - specify the labels that you want to use for NER. Defaults to the medical labels ["Medication/Vaccine", "MedicalProcedure", "AnatomicalStructure", "Symptom", "Disease"]
```

Comment:
Let's avoid specifying static labels on the instance level, since we can use whatever labels specified by the user in Label Studio project configuration. The suggested modifications in PR aim to use these labels dynamically